RecoHI/HiJetAlgos/python/hiFJGridEmptyAreaCalculator_cff.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+hiFJGridEmptyAreaCalculator = cms.EDProducer('HiFJGridEmptyAreaCalculator',
+                                 gridWidth = cms.untracked.double(0.05),
+                                 bandWidth = cms.untracked.double(0.2),
+                                 mapEtaEdges = cms.InputTag('hiFJRhoProducer','mapEtaEdges'),
+                                 mapToRho = cms.InputTag('hiFJRhoProducer','mapToRho'),
+                                 mapToRhoM = cms.InputTag('hiFJRhoProducer','mapToRhoM'),
+                                 pfCandSource = cms.InputTag('particleFlowTmp'),
+                                 jetSource = cms.InputTag('kt4PFJets'),
+								 doCentrality = cms.untracked.bool(True),
+								 hiBinCut = cms.untracked.int32(100),    
+								 CentralityBinSrc = cms.InputTag("centralityBin","HFtowers"),
+								 keepGridInfo = cms.untracked.bool(False),
```

Comment:
Do not use untracked parameters for parameters that affect physics quantities and results. It looks like all these untracked parameters should be changed, including those in hiFJRhoProducer.py.